USAGE.md

```diff
@@ -76,24 +122,24 @@ sed -i 's/VAULT_TOKEN=.*$/VAULT_TOKEN='"$token"'/g' docker-compose.yaml
 Unseal Vault:
 ```bash
 sealkey=$(cat vault-keys.json | jq -r ".unseal_keys_b64[]")
-docker exec -it vault vault operator unseal $sealkey
+sudo ${CONTAINER_RUNTIME} exec -it vault vault operator unseal $sealkey
 ```
 
 Remove Terraform container and recreate with Vault Token from previous step:
 ```bash
-docker-compose rm terraform
-docker-compose up -d terraform
+sudo ${CONTAINER_RUNTIME}-compose down terraform
```

Comment:
That would be `stop` here not `down` (https://docs.podman.io/en/latest/markdown/podman-stop.1.html)?

Reply:
okay, was `rm` and I wasn't sure what was right. Good catch!

Reply:
```suggestion
sudo ${CONTAINER_RUNTIME}-compose rm terraform
```

Reply:
```
$ sudo docker-compose stop -h
Stop running containers without removing them.

They can be started again with `docker-compose start`.

Usage: stop [options] [--] [SERVICE...]

Options:
  -t, --timeout TIMEOUT      Specify a shutdown timeout in seconds.
                             (default: 10)
$ sudo docker-compose rm -h
Removes stopped service containers.

By default, anonymous volumes attached to containers will not be removed. You
can override this with `-v`. To list all volumes, use `docker volume ls`.

Any data which is not in a volume will be lost.

Usage: rm [options] [--] [SERVICE...]

Options:
    -f, --force   Don't ask to confirm removal
    -s, --stop    Stop the containers, if required, before removing
    -v            Remove any anonymous volumes attached to containers
    -a, --all     Deprecated - no effect.
```
Please clarify what needs to happen here. afaiu `stop` just stops the container and `rm` stops **and** removes it. Is `rm` required or would `stop` be enough for docker and podman?

Reply:
```suggestion
sudo ${CONTAINER_RUNTIME}-compose stop terraform
```

Reply:
We can also use `stop` here, because we don't change the configuration of the container `terraform` :+1: